shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/queryable/ConvertYamlConfigurationHandlerTest.java

```diff
@@ -0,0 +1,136 @@
+        featureMap.put(sharding, shardingFilePath);
+    }
+    
+    @Before
+    public void init() {
+        ContextManager contextManager = mock(ContextManager.class, RETURNS_DEEP_STUBS);
+        ProxyContext.init(contextManager);
+    }
+    
+    @Test
+    public void assertExecuteWithAddResource() throws SQLException {
+        assertExecute(resource, resourceExpectedFilePath);
+    }
+    
+    @Test
+    public void assertExecuteWithCreateSharding() throws SQLException {
+        assertExecute(sharding, shardingExpectedFilePath);
+    }
+    
+    public void assertExecute(final String feature, final String expectedFilePath) throws SQLException {
+        ConvertYamlConfigurationHandler handler = new ConvertYamlConfigurationHandler();
+        handler.init(new ConvertYamlConfigurationStatement(Objects.requireNonNull(ConvertYamlConfigurationHandlerTest.class.getResource(featureMap.get(feature))).getPath()),
+                mock(ConnectionSession.class));
+        assertQueryResponseHeader((QueryResponseHeader) handler.execute());
+        assertTrue(handler.next());
+        assertRowData(handler.getRowData().getData(), expectedFilePath);
+        assertFalse(handler.next());
+    }
+    
+    private void assertQueryResponseHeader(final QueryResponseHeader actual) {
+        assertThat(actual.getQueryHeaders().size(), is(1));
+        assertQueryHeader(actual.getQueryHeaders().get(0));
+    }
+    
+    private void assertQueryHeader(final QueryHeader actual) {
+        assertThat(actual.getSchema(), is(""));
+        assertThat(actual.getTable(), is(""));
+        assertThat(actual.getColumnLabel(), is("distsql"));
+        assertThat(actual.getColumnName(), is("distsql"));
+        assertThat(actual.getColumnType(), is(1));
+        assertThat(actual.getColumnTypeName(), is("CHAR"));
+        assertThat(actual.getColumnLength(), is(255));
+        assertThat(actual.getDecimals(), is(0));
+        assertFalse(actual.isSigned());
+        assertFalse(actual.isPrimaryKey());
+        assertFalse(actual.isNotNull());
+        assertFalse(actual.isAutoIncrement());
+    }
+    
+    private void assertRowData(final Collection<Object> actual, final String expectedFilePath) {
+        assertThat(actual.size(), is(1));
+        assertThat(actual.iterator().next(), is(loadExpectedRow(expectedFilePath)));
+    }
+    
+    @SneakyThrows(IOException.class)
+    private String loadExpectedRow(final String expectedFilePath) {
+        StringBuilder result = new StringBuilder();
+        String fileName = Objects.requireNonNull(ConvertYamlConfigurationHandlerTest.class.getResource(expectedFilePath)).getFile();
+        try (
+                FileReader fileReader = new FileReader(fileName);
+                BufferedReader reader = new BufferedReader(fileReader)) {
+            String line;
+            while (null != (line = reader.readLine())) {
+                if (!line.startsWith("#") && !"".equals(line.trim())) {
```

Comment:
`"".equals(line.trim())` could be replaced to `Strings.isNullOrEmpty(line)`

Reply:
fixed~